x-pack/solutions/security/plugins/security_solution/common/detection_engine/prebuilt_rules/diff/extract_rule_data_query.ts

```diff
@@ -99,6 +99,9 @@ const normalizeFilterArray = (filters: RuleFilterArray | undefined): RuleFilterA
       ? {
           negate: filter.meta.negate,
           disabled: filter.meta.disabled !== undefined ? filter.meta.disabled : false,
+          params: filter.meta.params,
+          relation: 'relation' in filter.meta ? filter.meta?.relation : undefined,
+          type: filter.meta.type,
```

Comment:
What is this `type` field used for @nikitaindik? Didn't see it elsewhere in this PR. I was testing the original bug repros and found that this was one of the fields that propagates for all existing filters when you modify one of them. 
![Screenshot 2025-06-10 at 5 38 33 PM](https://github.com/user-attachments/assets/7f0ae8cb-3ce5-4437-85c6-921341ac2ef5)